awx/main/migrations/0059_v350_organization_max_hosts.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-06 15:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0058_v350_remove_limit_limit'),
```

Comment:
FYI, decent chance this is going to merge soon: https://github.com/ansible/awx/pull/3186/files#diff-8a5cc29c38894e8e792e1b4172465788R9

Reply:
yeah, figures.  I'll keep an eye on it.